lib/Driver/Driver.cpp

```diff
@@ -2031,6 +2031,12 @@ const ToolChain *Driver::getToolChain(const ArgList &Args) const {
       TC = new toolchains::Darwin(*this, Target);
       break;
     case llvm::Triple::Linux:
+      if (Target.isAndroid()) {
```

Comment:
@gribozavr [originally commented](https://github.com/apple/swift/pull/1442#discussion_r54128917) that an Android check may not be appropriate in a `GenericUnix` toolchain. I interpreted that to mean I should split Android out into its own toolchain, but I agree with you here, @compnerd. I'll rename `GenericUnix` to `Linux`, then include the `isAndroid()` check. Does that sound reasonable, @gribozavr?